tidb-in-kubernetes.md

```diff
@@ -0,0 +1,16 @@
+---
+title: 在 Kubernetes 上部署 TiDB
+summary: 了解如何在 Kubernetes 上部署 TiDB
+---
+
+# 在 Kubernetes 上部署 TiDB
+
+你可以使用 [TiDB Operator](https://github.com/pingcap/tidb-operator) 在 Kubernetes 上部署 TiDB。TiDB Operator 是 Kubernetes 上的 TiDB 集群自动运维系统，提供包括部署、升级、扩缩容、备份恢复、配置变更的 TiDB 全生命周期管理。借助 TiDB Operator，TiDB 可以无缝运行在公有云或私有部署的 Kubernetes 集群上。
+
+对于当前 TiDB 版本，TiDB Operator 工具的适用版本包括 v1.0、v1.1、v1.2。如无特殊需求，建议使用 TiDB Operator 的最新稳定版本 v1.2。
+
+TiDB Operator 的文档目前独立于 TiDB 文档。要查看如何使用 TiDB Operator 在 Kubernetes 上部署 TiDB 的详细步骤，请参阅对应版本的 TiDB Operator 文档：
+
+- [TiDB Operator v1.0 文档](https://docs.pingcap.com/zh/tidb-data-migration/v1.0/)
+- [TiDB Operator v1.1 文档](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/)
+- [TiDB Operator v1.2 文档](https://docs.pingcap.com/zh/tidb-data-migration/stable)
```

Comment:
```suggestion
- [TiDB Operator v1.2 文档](https://docs.pingcap.com/zh/tidb-data-migration/stable)
- [TiDB Operator v1.1 文档](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/)
- [TiDB Operator v1.0 文档](https://docs.pingcap.com/zh/tidb-data-migration/v1.0/)
```

Reply:
The links are for DM.

Reply:
Should we also add the version mapping as we do in https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/tidb-operator-overview?

Reply:
Because TiDB document is release-specific, we do not need to list the support information of all TiDB releases in this document. For each TiDB release, we could just let users know the corresponding supported TiDB Operator version.